### PR TITLE
feat: add connectors bundle example

### PR DESCRIPTION
**Pull request by chillleader**
## Description

Transfers the Bash Bundle example to this repository (previously was managed in the camunda/connectors-bundle repo)

## Related issues

closes camunda/team-connectors#258

### CODE REVIEW

**Comment by tmetzke** [comment]
This repository can be released completely independently from the Connectors Bundle, right? So we can release the base artifacts first with SDK and Connectors Bundle, and release over here afterward with the latest versions? Makes sense to me then 👍

File: runtime/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+> conflicts. To prevent this, common dependencies like `jackson` can be shaded and
+> relocated inside the connector jar. 
+> 
+> An example of such relocation with`maven-shade-plugin` can be found in the 
+> [Camunda 8 out-of-the-box Connectors repository](https://github.com/camunda/connectors-bundle/blob/cb925c1c5bdab9f55b77d2378be32116dec6f010/connectors/pom.xml#L83).
+
+Example adding a Connector JAR by extending the image
+
+```dockerfile
+FROM camunda/connectors:0.3.0
```
Comment (igpetrov) [inline runtime/README.md:19]: Shall it be a newer version?

```suggestion
FROM camunda/connectors:0.4.2
```

File: runtime/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+ADD https://repo1.maven.org/maven2/io/camunda/connector/connector-http-json/0.11.0/connector-http-json-0.11.0-with-dependencies.jar /opt/app/
+```
+
+Example adding a Connector JAR by using volumes
+
+```bash
+docker run --rm --name=connectors -d -v $PWD/connector.jar:/opt/app/ camunda/connectors:0.3.0
```
Comment (igpetrov) [inline runtime/README.md:27]:
```suggestion
docker run --rm --name=connectors -d -v $PWD/connector.jar:/opt/app/ camunda/connectors:0.4.2
```

File: runtime/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+For example, you can inject secrets when running a container:
+
+```bash
+docker run --rm --name=connectors -d \
+           -v $PWD/connector.jar:/opt/app/ \  # Add a connector jar to the classpath
+           -e MY_SECRET=secret \              # Set a secret with value
+           -e SECRET_FROM_SHELL \             # Set a secret from the environment
+           --env-file secrets.txt \           # Set secrets from a file
+           camunda/connectors:0.2.2
```
Comment (igpetrov) [inline runtime/README.md:42]:
```suggestion
           camunda/connectors:0.4.2
```

File: bundle/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Configure the connector runtime version
+ARG RUNTIME_VERSION=0.4.1
```
Comment (igpetrov) [inline bundle/Dockerfile:2]: I believe, the `0.4.2` is the latest one https://hub.docker.com/r/camunda/connectors/tags

```suggestion
ARG RUNTIME_VERSION=0.4.2
```

**Review by igpetrov — APPROVED** [review]
LGTM. Minor suggestions to bump versions.

**Review by igpetrov — APPROVED** [review]

**Pull request merged**